frontend/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:16.16.0-alpine3.16 AS build
+
+RUN npm install -g pnpm
+
+WORKDIR /app
+
+COPY package.json pnpm-lock.yaml ./
+RUN pnpm install
+
+COPY . .
+
+ARG API_ROOT http://localhost:8000/api
```

Comment:
Can you change the default value to https://tsmccn-backend.azurewebsites.net/api?

Reply:
I think we can remove the default value and set `API_ROOT` by passing the argument `--build-arg API_ROOT=<API_ROOT>`, so that we won't expose the backend API in our codebase, and that we don't have to modify our Dockerfile when the default API root changes

Reply:
There are several reasons why I hope to add the default value:

1. The backend API can still be found when using the frontend service even if we don't put it in the codebase.
2. The backend API is not supposed to change for a while.
3. It is more convenient for me to build the image every time the frontend changes.

Reply:
Oh, but I think put the backend API inside the codebase is a little bit non-elegant, so I think it is okay to keep the current version.